src/Runner.Worker/action_yaml.json

```diff
@@ -108,19 +108,26 @@
             }
         },
         "composite-steps": {
-            "context": [
```

Comment:
if the attribute needs a context, they'll have access (see "string-steps-context")